cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -61,6 +61,7 @@ const defaultState: AnnotationState = {
         collapsed: {},
         states: [],
         filters: [],
+        filtersHistory: [],
```

Comment:
In old UI we use local storage to save filters between sessions.
Let's do the same here.

Reply:
@bsekachev, maybe it can be useful to have one storage for user data like settings, history etc, can it?

Reply:
@ActiveChooN 

Don't think about settings storage now. It was not a feature before and we should not hasten about it. Need to discuss it IRL before implementation